ui_src/src/components/createStationForm/index.js

```diff
@@ -331,6 +345,29 @@ const CreateStationForm = ({ createStationFormRef, getStartedStateRef, finishUpd
                     </div>
                 </div>
             </div>
+            <div className="schema-type">
+                <Form.Item name="schemaValue">
+                    <div className="toggle-add-schema">
+                        <TitleComponent headerTitle="Use schema" typeTitle="sub-header"></TitleComponent>
+                        <Switcher onChange={() => setUseSchema(!useSchema)} checked={useSchema} disabled={schemas.length === 0} />
+                    </div>
+                    {!getStarted && schemas.length > 0 && useSchema && (
+                        <SelectSchema
+                            height="40px"
+                            width="450px"
+                            placeholder={creationForm.schemaValue || 'Select schema'}
+                            value={creationForm.schemaValue}
+                            colorType="navy"
+                            radiusType="semi-round"
+                            options={schemas}
+                            onChange={(e) => creationForm.setFieldsValue({ schemaValue: e })}
+                            // boxShadowsType="gray"
```

Comment:
You can only receive these props: {options, onChange, value, placeholder}. Please delete unnecessary fields.

Reply:
Done